package.json

```
@@ -122,6 +132,7 @@
   },
   "peerDependencies": {
     "@fastify/websocket": "^10 || ^11",
+    "crossws": "^0.3",
```

This will allow also for increments in the minor version (0.4, 0.5, etc.). Maybe this is expected or maybe you want to be stricter and use `~` instead.
```suggestion
    "crossws": "~0.3",
```